apex-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -50,6 +50,12 @@
       </includes>
     </fileSet>
     <fileSet filtered="true" encoding="UTF-8">
+      <directory>src/main/resources</directory>
+      <includes>
+        <include>**/log4j.props</include>
```

Comment:
When I named it as log4j.properties I was facing issue, and always hadoop log4j.properties was getting picked up. After your and david's comment, I tried updating classpath to include "current" folder first and it worked. 

So I will update PR to use log4j.properties and update classpath. @davidyan74  please check this and let me know if updating classpath might have any other side-effects.